inf_post/src/main.cpp

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+#include <string>
+#include <stack>
+
+#include "convert.h"
+#include "no_spaces.h"
+#include "checks.h"
+
+int main()
+{
+    try
+    {
+        std::string input;
+        getline(std::cin, input);
+        input = no_spaces(input);
+        if(!(is_valid(input)))
+        {
+            throw(input);
```

Comment:
The way you changed the code, now you don't need to throw exception. if it is not valid, just return with error

Reply:
+